doc/python/TableAPI.ipynb

```diff
@@ -11,7 +11,7 @@
   },
   {
    "cell_type": "code",
-   "execution_count": null,
+   "execution_count": 1,
```

Comment:
It's nicer to keep the Notebook clean of outputs :)

Reply:
Ouch, good catch. Thanks!